dashboard/modules/serve/serve_agent.py

```diff
@@ -3,8 +3,15 @@
 
 from aiohttp.web import Request, Response
 
+import dataclasses
+import ray
+import aiohttp.web
 import ray.dashboard.optional_utils as optional_utils
 import ray.dashboard.utils as dashboard_utils
+from ray.dashboard.modules.job.common import (
```

Comment:
The `serve_agent` shouldn't depend on `job`. Can we add a new file called `version.py` to the `ray/dashboard/modules/` directory (where `dashboard_sdk.py` is currently located) and move the `CURRENT_VERSION` and `VersionResponse` there? Then we can import from `version.py` in both the `job` directory and the `serve_agent.py` file.